pkg/apis/openebs.io/v1alpha1/cstorvolume_policy.go

```diff
@@ -38,6 +38,9 @@ type CStorVolumePolicy struct {
 
 // CStorVolumePolicySpec ...
 type CStorVolumePolicySpec struct {
+	// replicaAffinity is set to true then volume resources need to be
+	// distributed across the pool instances
+	ReplicaAffinity bool `json:"replicaAffinity"`
```

Comment:
Can it be under `Replica`?

Reply:
This doesn't go under Replica.. probably, new section like `VolumeConfig` or `Provisioner`?

Reply:
FIne with it

Reply:
we are not moving it to new section?